src/event-handler/src/use-cases/CreateEventUseCase.ts

```diff
@@ -19,7 +20,7 @@ export default class {
 
   async executeOnce(messageId: string, event: AuditLogEvent): PromiseResult<void> {
     if (!messageId) {
-      console.log(event)
+      logger.info(`No messageId: ${event}`)
```

Comment:
I think this doesn't log the event object and the output would be something like: `No messageId: [object Object]`